pulpcore/pulpcore/app/models/repository.py

```diff
@@ -105,6 +107,7 @@ def tls_storage_path(self, name):
     password = models.TextField(blank=True)
     last_synced = models.DateTimeField(blank=True, null=True)
     connection_limit = models.PositiveIntegerField(default=20)
+    policy = models.TextField(choices=POLICY_CHOICES, default=POLICY_MODE.IMMEDIATE)
```

Comment:
Is it possible for a plugin writer to override these fields somehow?

Reply:
They mostly could right now by defining the policy attribute on the subclass with another set of `choices`. There is one issue in that the helptext is duplicated because our use of choices isn't correct afaict. This PR kind of does the same thing since that problem already exists w/ the TASK_STATES. Other than that though they could do that, and we should fix that too but I'm not sure how yet. I filed it as this issue here:  https://pulp.plan.io/issues/4127

Note that these values affect the behavior of the streamer so if a plugin writer starts modifying the CHOICES values they are enabling/disabling features for the streamer. They need to be aware of that.

Reply:
Actually I'm removing this default because it doesn't work w/ DRF.